core/web-assets/src/main/assets/js/apps/onms-requisitions/lib/views/interface.html

```diff
@@ -25,20 +25,49 @@ <h3 class="modal-title">Interface</h3>
              ng-class="{ 'is-invalid' : intfForm.snmpPrimary.$invalid }">
       <p ng-show="intfForm.snmpPrimary.$invalid" class="invalid-feedback">A valid SNMP Primary Flag is required.</p>
     </div>
-    <label ng-show="requisitionInterface.services.length > 0">Services</label>
-    <ng-form name="serviceForm" ng-repeat="service in requisitionInterface.services">
-      <div class="form-group">
-        <div class="input-group">
-          <div class="input-group-prepend">
-            <div class="input-group-text">
-              <i class="fa fa-pencil"></i>
-            </div>
+
+    <hr>
+    <div class="clearfix">
+      <button type="button" class="btn btn-sm btn-secondary pull-right" id="add-metadata" ng-click="addMetaData()" ng-disabled="intfForm.$invalid">Add Meta-Data</button>
+      <h4>Meta-Data</h4>
```

Comment:
Can we add a way to collapse this? If there 10+ kvps, the modal get prretty long.

Reply:
Collapse added to meta-data and service list. I'm not sure about the right place for the custom CSS